radicale/storage.py

```diff
@@ -505,13 +505,38 @@ def update(self, href, vobject_item, etag=None):
     def delete(self, href=None, etag=None):
         # TODO: use etag in code and test it here
         # TODO: use returned object in code
+        def sync_directory(path):
```

Comment:
I feel like this would be better placed in another module within Radicale.

Maybe I should expose this in atomicwrites?